src/serve/hosting.ts

```diff
@@ -91,6 +91,7 @@ function startServer(options: any, config: any, port: number, init: TemplateServ
         firebaseMiddleware(req, res, next);
       },
     },
+    ...(options.frameworksDevModeHandle ? { after: { files: options.frameworksDevModeHandle } } : { }),
```

Comment:
Can this just be set as a variable above? It would be much easier to understand.